src/core/accido/_class_noun.py

```
@@ -5,9 +5,10 @@
 import logging
 from functools import total_ordering
 from typing import TYPE_CHECKING, Final, overload
+from warnings import deprecated
```

Pylint (3.3.7)

[error] No name 'deprecated' in module 'warnings'

(E0611)